src/Sylius/Bundle/AddressingBundle/Validator/Constraints/UniqueProvinceCollectionCodes.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\AddressingBundle\Validator\Constraints;
+
+use Symfony\Component\Validator\Constraint;
+
+final class UniqueProvinceCollectionCodes extends Constraint
+{
+    public string $message = 'sylius.country.unique_provinces_codes';
+
+    public function validatedBy(): string
+    {
+        return 'sylius_unique_province_collection_codes_validator';
+    }
```

Comment:
Similarly, as in the validator service's definition – we can get rid of this, and it'll work fine without it.